src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -255,8 +255,8 @@ protected override TaskMergeInstruction CanMergeWithCore(AsyncTask otherTask)
         private static IEnumerable<NodeModel> ComputeModifiedNodes(WorkspaceModel workspace)
         {
             var nodesToUpdate = new List<NodeModel>();
-            //Get those modified nodes that are not frozen
-            foreach (var node in workspace.Nodes.Where(n => n.IsModified && !n.IsFrozen))
+            //Get those modified nodes that are not frozen or preview states
+            foreach (var node in workspace.Nodes.Where(n => n.IsModified && !n.IsFrozen && !n.IsPreview))
```

Comment:
@mjkkirschner I think this line exclude the nodes from execution, right?

Reply:
How about if we modify the Nodes property so that it ignores the preview nodes in all contexts.
This way we can ensure that our preview nodes are not accounted for anywhere else.
And I guess we'll only account for them in all mechanisms engaged during preview placement.

Reply:
I'm not sure - I would take a look at all the places is `IsFrozen` is referenced - I kind of remember changes had to be made at the AST level so avoid the codegen step for frozen nodes - but it was a really long time ago.

Reply:
I think currently I covered all the place `IsFrozon` is referenced and relevant